students/pietrewicz_bartosz/lesson_02_flow_control/Lost_card.py

```diff
@@ -0,0 +1,10 @@
+# Read the number of cards
+N = int(input())
+# Read the numbers on cards and calculate their sum
+sum = 0
```

Comment:
Shadows built-in name 'sum'. Next time please avoid these names: https://docs.python.org/3/library/functions.html